src/main/java/transfer/countrytrans/CountryTransfer.java

```diff
@@ -0,0 +1,30 @@
+package transfer.countrytrans;
+
+import io.restassured.response.Response;
+import transfer.statetrans.StateTransfer;
+
+import static io.restassured.RestAssured.given;
+
+public class CountryTransfer extends StateTransfer {
+
+    public Response getByCountryState(String path, String country, String state) {
+        return given().pathParam("country", country).pathParam("state", state).spec(reqSpec.getBaseReqSpec()).when().get(path);
+    }
+
+    public Response getByText(String path, String text) {
+        return given().pathParam("text", text).spec(reqSpec.getBaseReqSpec()).when().get(path);
+    }
+
+    public Response getAll(String path) {
+        return given().spec(reqSpec.getBaseReqSpec()).when().get(path);
+    }
+
+    public Response getByIso2Code(String path, String alpha2Code) {
+        return given().pathParam("alpha2Code", alpha2Code).spec(reqSpec.getBaseReqSpec()).when().get(path);
```

Comment:
There is no need to have separate classes ByIso2 and ByIso3. Should be 1 class ByIso with rout = /get/{pram1}/{param2}

Reply:
changed